packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -449,6 +493,16 @@ class DescopeWc extends BaseDescopeWc {
       });
       return;
     }
+
+    this.logger().info(`Step "${stepName || `#${stepId}`}" is ${status}`, '', {
```

Comment:
I think we should better understand the usecase here, is it only for internal use or it will be relevant for the descoper, maybe this should be enabled only when in debug mode

In addition, I don't think it's a good practice to build the error string here, let's use a function for it

Reply:
not sure i folllow @nirgur, this will be used by descopers (also using console.log), this is not an error but a "debug" message to callback with the current flow state. The problem with debug mode is that it is tidily coupled with the debugger while to disable/enable only the debugger window is breaking changes. 


We should be able to let descopers decide how to consume the logger, currently we will provide console and log event.